packages/core/src/lib/Select/OldSelect/OldSelect.styles.ts

```diff
@@ -0,0 +1,237 @@
+import { LabelProps, SelectedItemProps, ISelectProps } from '../types';
+import { color, fonts, resetCSS } from '@web3uikit/styles';
+import styled, { css } from 'styled-components';
+
+const DivStyledWrapper = styled.div<Pick<ISelectProps, 'state'>>`
+    ${resetCSS};
+    display: inline-block;
+    position: relative;
+`;
+
+const LabelStyled = styled.label<LabelProps>`
+  ${resetCSS}
+  ${fonts.text}
+    background-color: ${color.white};
+  height: 24px;
+  left: ${({ hasPrefixIcon, hasSelectedIndex }) =>
+      hasPrefixIcon && !hasSelectedIndex ? 50 : 12}px;
+  line-height: 1;
+  pointer-events: none;
+  position: absolute;
+  padding: 0 4px;
+  transition: all 0.1s ease-out;
+  ${({ hasSelectedIndex }) =>
+      hasSelectedIndex ? 'top: -9px; font-size: 14px;' : 'top: 20px;'};
+`;
+
+const PrefixSpan = styled.span`
+    margin-right: px;
+`;
+
+const SelectedItem = styled.div<SelectedItemProps>`
+    ${resetCSS};
+    ${fonts.text};
+    background-color: ${color.white};
+    border-radius: 16px;
+    border: 2px solid;
+    cursor: pointer;
+    display: flex;
+    height: 56px;
+    overflow: hidden;
+    padding: 14px 50px 14px 16px;
+    transition: all 0.1s linear;
+
+    border-color: ${({ state }) => {
+        switch (state) {
+            case 'error':
```

Comment:
how can a Select be in error?

Reply:
maybe if it is required but missed by the user 🤔

Reply:
it can have error state for validation
not yet added in the new version.
will do a bit more research for this 🙌